src/Core/common/SetPixelValueInOneImage/Code.py

```diff
@@ -45,7 +45,7 @@
 image = ImageType.New()
 image.SetRegions(region)
 image.Allocate()
-image.FillBuffer(itk.NumericTraits[PixelType].Zero())
+image.FillBuffer(itk.NumericTraits[PixelType].Zero)
```

Comment:
This should also fail for [the same reason as above](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/174#discussion_r459119051).

Reply:
Changed to `.ZeroValue()` in 63d0bf6.